docs/test-vectors/IntentToSell.json

```
@@ -237,7 +237,7 @@
         "sku": "163443407321"
       },
       "declarationDate": "1-1-2021",
-      "sellByDate": "10-2-2021"
+      "sellByDate": "10-3-2021"
```

```suggestion
      "sellByDate": "10-3-2021"
```
I don't understand how changing dates falls under fixing IRIs ... but as long as you're fixing dates, please let's use ISO standard YYYY-MM-DD across the board!  (Is the above meant to say March 10, i.e., is it DD-MM-YYYY, or October 3, i.e., is it MM-DD-YYYY?)

---

per README:

> Wherever possible, dates should be formatted as YYYY-MM-DD so as to be directly compatible with xsd:date.

---

This stuff is all autogenerated... please open an issue to track this, I am not going to fix anything but links in this PR.

---

...and yet, multiple dates have had their middle-element increased by 1.

Fine, new issue coming.